instrumentation/rails/lib/opentelemetry/instrumentation/rails/instrumentation.rb

```diff
@@ -13,34 +13,29 @@ module Rails
       # instrumentation
       class Instrumentation < OpenTelemetry::Instrumentation::Base
         install do |_config|
+          OpenTelemetry::Instrumentation::ActiveRecord::Instrumentation.instance.install({})
+          OpenTelemetry::Instrumentation::ActionPack::Instrumentation.instance.install({})
```

Comment:
What kind of final, end-state did you have in mind?

Personally I think we should go one of two ways:
- Install all rails-related gems by default (and provide an option that allows you to selectively turn them off)
- Install no rails-related gems by default (and provide an option that allows you to choose which to enable)

Of those two, I would actually prefer the first one. My rationale, over all, is that we probably don't want to be in the business of telling people precisely which parts of rails they want instrumented, because we can't really make a good judgement call there. For example, why ActiveRecord and not ActionView? (Not that this is a particularly bad choice you've made, just using it as an example).

Reply:
> Of those two, I would actually prefer the first one.

I'm glad, that's the direction I had planned for this :) 

> My rationale, over all, is that we probably don't want to be in the business of telling people precisely which parts of rails they want instrumented, because we can't really make a good judgement call there.

Yup, let people turn off the bits they are not interested in.